contracts/contracts/token/OUSD.sol

```diff
@@ -242,11 +293,38 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         uint256 creditsCredited = _value.mulTruncate(_creditsPerToken(_to));
         uint256 creditsDeducted = _value.mulTruncate(_creditsPerToken(_from));
 
-        _creditBalances[_from] = _creditBalances[_from].sub(
-            creditsDeducted,
-            "Transfer amount exceeds balance"
-        );
-        _creditBalances[_to] = _creditBalances[_to].add(creditsCredited);
+        if (rebaseState[_from] == RebaseOptions.Delegator) {
+            address delegatee = yieldDelegateFrom[_from];
+            // adjust delegator and delegatee fixed amounts
+            _amountAdjustement[_from] -= _value;
```

Comment:
I did some a few code explorations here yesterday. I think rather than plussing or subtracting the values, we should recompute based on balance every time. In this way we no rounding errors build up over time.